enarx-keep-sgx-shim/src/main.rs

```diff
@@ -23,3 +23,20 @@ mod entry;
 mod event;
 mod handler;
 mod libc;
+
+use span::Line;
+
+// NOTE: this must be kept in sync with enarx-keep-sgx
```

Comment:
I'd much rather branch it into a separate library consumed by both. I did it this way for now because I suspect this needs `cargo make`. Also, `enarx-keep-sgx` uses `usize` and `enarx-keep-sgx-shim` uses `u64`. That's also a bigger strategy problem that boils down to the `StateSaveArea` struct definition. I think the right solution is probably adding some debt bugs.

Reply:
I added a debt bug: https://github.com/enarx/enarx/issues/424